texar/torch/modules/classifiers/conv_classifiers.py

```diff
@@ -166,6 +173,11 @@ def forward(self,  # type:ignore
                 first be masked out before feeding to the layers.
             dtype (optional): Type of the inputs. If not provided, infers
                 from inputs automatically.
+            data_format (optional): Data type of the input tensor. If
+                ``channels_last``, the last dimension will be treated as channel
+                dimension. If ``channels_first``, first dimension will be
+                treated as channel dimension. Defaults to None.
+                If None, the value will be picked from hyperparameters.
```

Comment:
Make the input size clearer, e..g, 

If ``channels_last``, the last dimension will be treated as channel dimension, so the size of :attr:`input` should be `[batch_size, X, channel]` , ...

Reply:
Updated in d2c2634